internal/plugin/validators/lua_validator_test.go

```diff
@@ -1645,10 +1645,10 @@ func TestLuaValidator_GetDB(t *testing.T) {
 	storeLoader := setupStoreLoader(t)
 	require.NotNil(t, storeLoader)
 	goodValidator.storeLoader = storeLoader
-	t.Run("context not containing RequestCluster returns error", func(t *testing.T) {
+	t.Run("context not containing RequestCluster returns default store", func(t *testing.T) {
 		db, err := goodValidator.getDB(context.Background())
-		assert.Nil(t, db)
-		assert.Error(t, err, "invalid context: failed to retrieve RequestCluster from context")
+		require.NoError(t, err)
+		assert.NotNil(t, db)
```

Comment:
We should likely track some work to expose a `Cluster() string` method on the DB, so we can actually assert this better in tests... as well, no other way to check without the use of nasty reflection.

Reply:
Implemented in Kong/koko#482.